waku/api.go

```diff
@@ -364,6 +367,8 @@ func (api *PublicWakuAPI) Messages(ctx context.Context, crit Criteria) (*rpc.Sub
 	}
 
 	for i, bt := range crit.Topics {
+		// TODO these checks will always be false, common.TopicType has a predefined length and so any other length
+		//  would cause a compile error.
 		if len(bt) == 0 || len(bt) > 4 {
 			return nil, fmt.Errorf("subscribe: topic %d has wrong size: %d", i, len(bt))
 		}
```

Comment:
These checks will always be false, `common.TopicType` has a predefined length and so any other length would cause a compile error.

Any objection to removing these checks?

Reply:
Yeah, if compiler can catch it no reason for us to check for it.